pycqed/analysis/tomography_toolbox.py

```diff
@@ -0,0 +1,546 @@
+import qutip as qt
+import numpy as np
+import time
+import scipy
+import os
+from analysis import pytomo as csdp_tomo
+import uuid
+import glob
+
+
+comp_projectors = [qt.ket2dm(qt.tensor(qt.basis(2,0), qt.basis(2,0))),
+                  qt.ket2dm(qt.tensor(qt.basis(2,0), qt.basis(2,1))),
+                  qt.ket2dm(qt.tensor(qt.basis(2,1), qt.basis(2,0))),
+                  qt.ket2dm(qt.tensor(qt.basis(2,1), qt.basis(2,1)))]
+
+class TomoAnalysis():
```

Comment:
How does this file relate to the other (execute tomo and mav2.tomography object) tomography files?